loc/npcs/trader.js

```diff
@@ -61,7 +61,7 @@ export default {
     bg: 'Продайте предметите си',
     cs: 'Prodávejte své zboží',
     pl: 'Sprzedawaj swoje przedmioty',
-    es: 'Vende sus ítemes',
+    es: 'Vende tus articulos',
```

Comment:
is articulos better than itemes her? what are the meanings?

Reply:
I pointed this out yesterday regarding a different file, there is no word for "item" in Spanish. "articulos" is the closest you can get. Verified by two people.

Reply:
hmmm dek devy sincerelly i don't know i pushed in translate google and it says items = Articulos :uuuuu

Reply:
dek ''Articulos'' in english means Articles and ''items'' not have means

Reply:
So i don't know means ''items''

Reply:
items maybe means ''Conjunto''
.-.

Reply:
or item's maybe too means ''Elementos''

Reply:
I think what items means ''elementos''